src/services/AuthService.ts

```diff
@@ -0,0 +1,14 @@
+import { Service } from '@tsed/common';
```

Comment:
Keep in mind that every request other than login/sign NEEDS to check if the JWT is valid. I'll approve this PR because the code is okay, but the ticket is not completely finished.